src/RSCPL.xml

```diff
@@ -0,0 +1,473 @@
+<SPDX name="Ricoh Source Code Public License" identifier="RSCPL" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/RSCPL</url>
+  </urls>
+  <license>
+    <title>
+      <p>Ricoh Source Code Public License 
+        <br/>Version 1.0 
+      </p>
+    </title>
+
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions.</p>
+
+      <list>
+        <li>
+          <b>1.1.</b>
+          <p>"Contributor" means each entity that creates or contributes to the creation of Modifications.</p>
+        </li>
+        <li>
+          <b>1.2.</b>
+          <p>"Contributor Version" means the combination of the Original Code, prior Modifications used by a
+             Contributor, and the Modifications made by that particular Contributor.</p>
+        </li>
+        <li>
+          <b>1.3.</b>
+          <p>"Electronic Distribution Mechanism" means a website or any other mechanism generally accepted in
+             the software development community for the electronic transfer of data.</p>
+        </li>
+        <li>
+          <b>1.4.</b>
+          <p>"Executable Code" means Governed Code in any form other than Source Code.</p>
+        </li>
+        <li>
+          <b>1.5.</b>
+          <p>"Governed Code" means the Original Code or Modifications or the combination of the Original Code
+             and Modifications, in each case including portions thereof.</p>
+        </li>
+        <li>
+          <b>1.6.</b>
+          <p>"Larger Work" means a work which combines Governed Code or portions thereof with code not
+             governed by the terms of this License.</p>
+        </li>
+        <li>
+          <b>1.7.</b>
+          <p>"Licensable" means the right to grant, to the maximum extent possible, whether at the time of the
+             initial grant or subsequently acquired, any and all of the rights conveyed herein.</p>
+        </li>
+        <li>
+          <b>1.8.</b>
+          <p>"License" means this document.</p>
+        </li>
+        <li>
+          <b>1.9.</b>
+          <p>"Modifications" means any addition to or deletion from the substance or structure of either the
+             Original Code or any previous Modifications. When Governed Code is released as a series of
+             files, a Modification is:</p>
+
+      <list>
```

Comment:
This (and some of the subsequent lines) could use deeper indentation to show that they're part of the 1.9 list entry.